pkg/services/accesscontrol/migrator/zanzana.go

```diff
@@ -33,6 +34,8 @@ func NewZanzanaSynchroniser(client zanzana.Client, store db.DB, collectors ...Tu
 		collectors,
 		teamMembershipCollector(store),
 		managedPermissionsCollector(store),
+		folderParentCollector(store),
```

Comment:
so we only collect the "parent" folders?

Reply:
yes, we don't need root folders here

Reply:
We only need to write relations if folder is a child of another one. So in case of `A -> B -> C` we need to write 2 tuples:
```
folder:A parent folder:B
folder:B parent folder:C
```

So no explicit mention of folder:A having no parent required.